charts/provider-metal/templates/configmap.yaml

```diff
@@ -30,6 +30,8 @@ data:
     auth:
       enabled: {{ .Values.config.auth.enabled }}
       providerTenant: {{ .Values.config.auth.providerTenant }}
+    audit:
```

Comment:
I think it would be better to explicitly call it `splunkAuditWebhook` or something like this just to make sure that this is a really specific thing only working for Splunk.

Reply:
Renamed audit to splunkAudit where it seemed appropriate.